docs/learn/chrysalis/core-concepts/switch-to-UTXO.md

```diff
@@ -1,58 +1,74 @@
 ---
-sidebarLabel: Switch to UTXO
-description: Learn about the transformation from balance models to Unspent Transaction Output (UTXO) models in IOTA, outlining the problems, solutions, and implications of this change.
-keywords:
-  [
-    'IOTA',
-    'UTXO',
-    'Balance Models',
-    'Conflicts',
-    'Reattachments',
-    'Colored Coins',
-  ]
+description: This article provides an in-depth look into IOTA's shift from WOTS addresses to the UTXO model and the motivations and processes behind this migration.
+keywords: [ 'IOTA', 'WOTS', 'UTXO', 'Migration', 'Chrysalis', 'Ed25519', 'Security', 'Protocol Design', 'User Experience' ]
 ---
+# Switch to UTXO 
 
-# The Switch to UTXO in IOTA
+With Chrysalis, IOTA moved away from the Winternitz One-Time Signature (WOTS) system to the Unspent Transaction Output (
+UTXO) model, addressing several issues related to security, protocol design, and user experience.
 
-Before Chrysalis, IOTA used an account model where every address had a single value, its current balance. However,
-as issues arise when handling conflicts (i.e., double spends), it becomes difficult to ascertain which transactions are
-legitimate and which ones are effectively double spends.
+## Challenges with the Account Balance Model
 
-### The Challenge with Balance Models
+Before the transition to Chrysalis, IOTA operated on an account model.
+\Every address held a distinct balance value.
+However, when double-spends or conflicts arose, discerning legitimate transactions from double-spends became complex.
+This model posed two main challenges:
 
-The balance model is relatively straightforward - it keeps track of every address and its corresponding balance.
-However, this simplicity poses some significant problems:
+- **Conflict Handling**: Identifying the transaction performing a double-spend was intricate, leading to extensive
+  "conflict sets" and reducing the efficiency in addressing conflicts.
+- **Reattachments**: In cases where funds were received at an already spent address, it became possible for anyone to
```

Comment:
I'm actually not sure if that is true? I thought the problem was somehow exposing your private key with every signature of an already spend address or something like that 🤔.